protocols/bftcosi/bftcosi.go

```diff
@@ -15,13 +14,16 @@ import (
 // VerificationFunction can be passes to each protocol node. It will be called
 // (in a go routine) during the (start/handle) challenge prepare phase of the
 // protocol. The passed message is the same as sent in the challenge phase.
-type VerificationFunction func(Msg []byte) bool
+type VerificationFunction func(Msg []byte, Data []byte) bool
```

Comment:
Probably not the nicest thing to do. The `Msg` is signed and verified, and the `Data` is just sent along. So I don't have to send another message before it. Now that I think of it: do we have a working `configuration`-message for services the way we discussed some time ago? I could send the `Data` along there.